client/src/scenes/books/Books.tsx

```diff
@@ -1,9 +1,39 @@
+import BookCard from './BookCard'
+import BookService from '../../services/books.service'
 import './Books.scss'
+import { useEffect, useState } from 'react'
+import { BookType } from 'types/books.types'
 
 const Books = () => {
+  const [books, setBooks] = useState<BookType[]>([])
+  const [step, setStep] = useState(1)
+  const [itemsEachStep] = useState(30)
+
+  useEffect(() => {
+    BookService.getBooks().then((res) => {
+      setBooks(res.data)
+    })
+  }, [])
+
+  const handleShowMore = () => {
+    setStep(step + 1)
+  }
   return (
     <div className='books'>
-      <p>List of books here</p>
+      <div className='book-list'>
+        {books.slice(0, itemsEachStep * step).map((book, index) => (
+          <BookCard item={book} key={index} />
+        ))}
+      </div>
+      {books.length > 30 && (
+        <button
+          data-testid='show-more-btn'
+          className='show-more-btn'
+          onClick={handleShowMore}
+        >
+          More Product
+        </button>
```

Comment:
Show more is another feature that should be implemented later since the story hasn't been mentioned.
- This more-product feature will depend on a limit parameter from the server. When users click more-product, the API will call again with a new range. 
- We shouldn't fetch all books at 1 time and then slice them in FE. This way will affect performance. Imagine we have 100000 books, then the request will take a lot of time.